docs/glossary.md

```diff
@@ -0,0 +1,132 @@
+
+### URN 
+
+[URN](https://en.wikipedia.org/wiki/Uniform_Resource_Name) stands for  Uniform Resource Name, they are used as global unique identifiers for assets across the Metaverse projects. 
+
+- URN Resolver: https://github.com/decentraland/urn-resolver
+- URN Namespace: https://github.com/common-metaverse/urn-namespaces
+   
+### Content Server
+
+The [Content Server](https://github.com/decentraland/catalyst/tree/main/content) is a part of the Catalyst node in charge of managing the Decentralized storage of Entities Deployments. Currently, all content stored in a Content Server is synchronized with the rest of the DAO Catalysts.
+
+### Lambdas
+
+The [Lambdas](https://github.com/decentraland/catalyst/tree/main/lambdas) Service is a part of the Catalyst node that that works as a reconciliation layer between the Content Servers and the Blockchain, sanitizing content, validating ownership and helping retrieve information for the Catalyst clients.
+
+### Synchronization 
+
+Content Servers have to offer the ability to deploy content on them, and also ensure that any content deployed on any other valid and running Catalyst from the DAO List is available there too. Resolving this is called **synchronization**. 
+
+### Content SNAPSHOT 
+
+A set of all the  Active Entities of the specified type. This Content SNAPSHOTS are used to synchronize content between Catalyst nodes.
+
+### Realm 
+
+When connected to Decentraland, you would only be able to interact with Peers connected to the same Realm. As friendly-names Realms make use of Greek Gods names such as Hela or Loki. Each realm is determined by a single [Catalyst Node](https://decentraland.github.io/catalyst-monitor/).
+
+### Peer 
+
+A user connected to Decentraland. 
+
+### Lighthouse 
+
+The [Lighthouse](https://github.com/decentraland/lighthouse) is the name used t reference the previous version of the Communications Service hosted on the Catalyst nodes and used to group users in a P2P network. 
+Main responsibilities were to determine which are the candidates for a P2P connection and do the WebRTC signaling to establish the connection. Most of this logic was done through two external components: the PeerJS Server (connects [WebRTC](https://webrtc.org/) peers) and Archipelago (receives users positions and groups them in islands).
+
+### Backend for Frontend (aka BFF) 
+
+The [BFF](https://github.com/decentraland/explorer-bff) Service was created to resolve client needs and to enable faster development of new features without breaking the existing APIs. In the context of the Catalyst Communication Service, it will be in charge of managing the P2P signaling and thus replacing the Lighthouse.
+
+### Catalyst Owner
+
+A [Catalyst Owner](https://github.com/decentraland/catalyst-owner) is an individual or organization running a Catalyst node that is part of the DAO. To add a Catalyst node to the DAO network a [proposal](https://governance.decentraland.org/?type=catalyst) needs to be submitted.
```

Comment:
I think a catalyst-owner doesn't really need to be part of the DAO (although of course that's the only way to really contribute to the network right now)